addons/account/controllers/portal.py

```diff
@@ -16,9 +16,13 @@ class PortalAccount(CustomerPortal):
     def _prepare_home_portal_values(self, counters):
         values = super()._prepare_home_portal_values(counters)
         if 'invoice_count' in counters:
-            invoice_count = request.env['account.move'].search_count(self._get_invoices_domain()) \
+            invoice_count = request.env['account.move'].search_count(self._get_invoices_domain('in'), limit=1) \
```

Comment:
Wait if we're doing limit=1 won't it always be 0 or 1?

Reply:
> Wait if we're doing limit=1 won't it always be 0 or 1?
Cards were changed to not show number anymore, (not for all but most of them) if there is at least 1 invoice/bill the card will be shown so there is no point in knowing there is more.

Limit could be added everywhere but for the sake of (finally) merging this PR I only changed where I made changes.

Reply:
Makes sense, thanks